build.sbt

```diff
@@ -127,13 +137,12 @@ lazy val seleniumJSEnvTest: Project = project.
 lazy val seleniumJSHttpEnvTest: Project = project.
   enablePlugins(ScalaJSPlugin).
   enablePlugins(ScalaJSJUnitPlugin).
-  settings(testSettings).
+  settings(httpTestSettings)
+
+lazy val seleniumJSHttpESModuleEnvTest: Project = project.
```

Comment:
You don't necessarily need to create a new project for this. Everything that is tested in `seleniumJSHttpEnvTest` can also be tested in `ESModule` mode.

Either works for me, but if you want the separate project, you should probably materialize it in a different server directory.

Also, it needs to be added to `.travis.yml`.

Reply:
Ok, happy with a single project with ESModule setting on :)